### ISSUE TITLE
file name for DBG and PRT output files

### ISSUE THREAD

**Issue by GitPaean** [OPM/opm-common#532]
I noticed recently that from time to time I got file names like follows, while not all the time,

```
NORNE_ATW2013..DBG  NORNE_ATW2013..PRT 
```

In some different runnings
```
NORNE_ATW2013.DBG NORNE_ATW2013.PRT
```

Any explanations for this? The first time when I saw this, I thought it might be something intentional, while it is not always the case.

**Comment by GitPaean** [comment]
> Were any of these parallel runs using MPI?

Nope. I compiled the code with MPI, while I did not use `mpirun` at all.

**Comment by bska** [comment]
> > Were any of these parallel runs using MPI?
> 
> Nope. I compiled the code with MPI, while I did not use `mpirun` at all.

Okay, then I'm out of ideas because I'm not seeing that symptom in my local runs.  My only other suggestion, if this is somewhat reproducible for you, would be to either put a debugging breakpoint or some local printing to `setupLogging()` in [`FlowMain.hpp`](https://github.com/OPM/opm-simulators/blob/33da0aea09158408563a84393400f08efd41c38a/opm/autodiff/FlowMain.hpp#L392) and/or [`FlowMainEbos.hpp`](https://github.com/OPM/opm-simulators/blob/33da0aea09158408563a84393400f08efd41c38a/opm/autodiff/FlowMainEbos.hpp#L324) to try to tease out where the extra full stop character arises.

**Comment by GitPaean** [comment]
Thanks. I might try to figure it out later, while it does not bother me much. 

Just curious whether other people also see this.

**Comment by blattms** [comment]
my wild guess: Are you maybe running "flow DECK." (omitting DATA). It would
still be a bug.

**Comment by GitPaean** [comment]
> my wild guess: Are you maybe running "flow DECK." (omitting DATA). It would
still be a bug.

It is not wild at all, you are totally correct. I have been always trying to use full `DECK.DATA` file name, while at some cases, there are more files under the folder (DECK.INI, DECK.PRT, etc.), with tab-auto-completion of file name, it can results in `DECK.` . I have never been aware of we can omit `.DATA`, so I did not realize it at all. 

```
NORNE_ATW2013..DBG  NORNE_ATW2013.EGRID  NORNE_ATW2013.INIT  NORNE_ATW2013..PRT  NORNE_ATW2013.SMSPEC  NORNE_ATW2013.UNRST  NORNE_ATW2013.UNSMRY
```
